tests/storage/online_resize/conftest.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+"""
+Fixtures for online resize tests
+"""
+
+import pytest
+
+from tests.storage.online_resize.utils import (
+    SMALLEST_POSSIBLE_EXPAND,
+    STORED_FILENAME,
+    cksum_file,
+    create_rhel_dv_from_data_source,
+    expand_pvc,
+    wait_for_resize,
+)
+from utilities.constants import OS_FLAVOR_RHEL, Images
+from utilities.storage import create_dv, is_snapshot_supported_by_sc
+from utilities.virt import VirtualMachineForTests, running_vm
+
+
+@pytest.fixture(scope="module")
+def skip_if_storage_for_online_resize_does_not_support_snapshots(
+    storage_class_matrix_online_resize_matrix__module__, admin_client
+):
+    sc_name = [*storage_class_matrix_online_resize_matrix__module__][0]
+    if not is_snapshot_supported_by_sc(
+        sc_name=sc_name,
+        client=admin_client,
+    ):
+        pytest.xfail(f"Storage class for online resize '{sc_name}' doesn't support snapshots")
+
+
+@pytest.fixture()
+def orig_cksum(rhel_vm_for_online_resize, running_rhel_vm):
+    return cksum_file(vm=rhel_vm_for_online_resize, filename=STORED_FILENAME, create=True)
+
+
+@pytest.fixture()
+def rhel_dv_for_online_resize(
+    request,
+    namespace,
+    unprivileged_client,
+    storage_class_matrix_online_resize_matrix__module__,
+    rhel10_data_source_scope_module,
+):
+    with create_rhel_dv_from_data_source(
+        unprivileged_client=unprivileged_client,
+        namespace=namespace.name,
+        name=request.param["dv_name"],
+        storage_class=[*storage_class_matrix_online_resize_matrix__module__][0],
+        rhel_data_source=rhel10_data_source_scope_module,
+    ) as dv:
+        yield dv
+
+
+@pytest.fixture()
+def second_rhel_dv_for_online_resize(rhel_dv_for_online_resize, unprivileged_client):
+    with create_dv(
+        source="pvc",
+        dv_name=f"{rhel_dv_for_online_resize.name}-target",
+        namespace=rhel_dv_for_online_resize.namespace,
+        client=unprivileged_client,
+        size=rhel_dv_for_online_resize.size,
+        storage_class=rhel_dv_for_online_resize.storage_class,
+        volume_mode=rhel_dv_for_online_resize.volume_mode,
```

Comment:
`volume_mode=rhel_dv_for_online_resize.volume_mode,` can be removed as we are not specifying the volume mode explicitly